tests/unittests/cli/test_cli.py

```diff
@@ -15,6 +15,7 @@
 CLI_ARGS = (
     f"--keystore-file {KEYSTORE_PATH.joinpath('UTC--1')} "
     f"--no-ui "
+    "--chain smoketest:http://localhost:12345 "
```

Comment:
is that necessary here? Or rather whats the purpose?

Reply:
https://github.com/raiden-network/scenario-player/pull/516/commits/beead0c0cc95518380f9014adffde95cdde682f3 makes the `--chain ...` argument mandatroy, so the cli tests don't run without it anymore